generated/user_existence/user_existence.json

```diff
@@ -0,0 +1,29 @@
+{
+  "primary_applicant": {
+    "id": "58bf3bb6e41b0738bf0000ce",
+    "user_id": "58bf3b87e41b0738bf0000cc",
+    "first_name": "Loren",
+    "last_name": "Dimallo"
+  },
+  "employers": [
+    {
+      "employer": {
+        "id": "58bf321ee41b0738bf00000c",
+        "legal_name": "BriCol",
+        "phone": "(783) 786-7857"
+      },
+      "broker": {
+        "id": "58c938e2e41b071e74000085",
+        "legal_name": "WilliamBroker",
```

Comment:
Legal name of the person with the broker role, if memory/understanding serves right. I suggest we create a Hackpad, make a list of all field changes and then, address it as part of a single task/PR in the interest of efficiency.